src/libraries/System.Linq.Expressions/tests/Dynamic/CallSiteCachingTests.cs

```diff
@@ -218,5 +222,63 @@ public void BinderCacheFlushWhenTooBig()
             Assert.NotEqual(rules1, rules3);
 
         }
+
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
```

Comment:
Outerloop?

Reply:
When are `[OuterLoop]` tests executed?

Reply:
By the developer explicitly locally, on request with an azp command on PRs, and daily-ish on rolling builds.

Reply:
> daily-ish on rolling builds

Thanks. In that case, could we keep the test as-is, in all runs? The test only takes ~0.2 secs to complete.

Reply:
Is that true even on underpowered machines, ARM, etc.? Looks a bit like a stress test. If it'll run that quickly everywhere, then innerloop is fine. Thanks.

Reply:
> Is that true even on underpowered machines, ARM, etc.?

I haven't found how to determine test execution time on ARM yet. If the execution time is too slow on some platforms, perhaps we could move this test to outer loop, or limit the test to a subset of platforms, in a subsequent PR.